samples/bookinfo/kube/istio-rbac-enable.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: "config.istio.io/v1alpha2"
+kind: authorization
+metadata:
+  name: requestcontext
+  namespace: istio-system
+spec:
+  subject:
+    user: source.user | ""
+    groups: source.user | ""
+    properties:
+      app: source.labels["app"] | ""
+      version: source.labels["version"] | ""
+      namespace: source.namespace | "default"
```

Comment:
Let's do `namespace: source.namespace | ""`

`default` is a real namespace in our examples, so we don't want it confused for an unknown namespace.